docs/ops/network-security.md

```diff
@@ -0,0 +1,27 @@
+# Network Security
+
+## Network Security Perimeter
+
+The Network Security Perimeter allows organizations to define a logical network isolation boundary for Platform as a Service (PaaS) resources, such as Azure Storage accounts and KeyVaults, that are deployed outside a virtual network. It restricts public network access to PaaS resources within the perimeter, but you can exempt access by using explicit access rules for public inbound and outbound traffic.
+
+EV2 does not support private endpoint connectivity. However we need direct access from EV2 to some key vaults in order to manage them. For this we want to find out if NSP [1] can be used for securing KeyVault access. For this we can leverage the list of public IPs [2] used by EV2 and secure the access to KeyVault with it, since it also supports service tags we can use them as well [4].
+
+## Private Endpoint
```

Comment:
can you add more details how layed down the private endpoints, e.g. into one of the subnets of the respective AKS clusters etc